editor/components/button/style.scss

```diff
@@ -1,8 +1,14 @@
 .editor-button {
 	background: none;
 	border: none;
+	outline: none;
 
 	&:disabled {
 		opacity: 0.6;
 	}
+
+	&.is-active {
```

Comment:
Maybe a question for @jasmussen for prescribing variations on button state we'll need to support.